packages/perseus-editor/src/components/interactive-graph-settings.tsx

```diff
@@ -80,11 +123,6 @@ class InteractiveGraphSettings extends React.Component<Props, State> {
     }
 
     static defaultProps = {
-        editableSettings: ["graph", "snap", "image", "measure"],
-        box: [
-            interactiveSizes.defaultBoxSizeSmall,
-            interactiveSizes.defaultBoxSizeSmall,
-        ],
```

Comment:
Removing `box` as a prop because it's only ever getting the small size from InteractiveGraphEditor. Instead, I'm hard coding it as the small box everywhere.